build/script.serve.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Serves built files from 'dist', proxies API requests and websockets to backend
+ */
+
+const express = require('express')
+const proxyMiddleware = require('http-proxy-middleware')
+const { join } = require('path')
+
+/**
+ * Serve static files
+ */
+const app = express()
+app.use(express.static(join(__dirname, '../dist')))
+
+// copied and adjusted from index.js
+const BACKEND = 'http://localhost:8000'
+const proxyTable = {
+  '/api': {
+    target: BACKEND,
+    changeOrigin: true,
+    ws: true,
+    onProxyReq: (proxyReq) => {
+      if (/^https:/.test(BACKEND)) {
+        proxyReq.setHeader("referer", BACKEND);
+      }
+    }
+  }
+}
+
+Object.keys(proxyTable).forEach(function (context) {
+  var options = proxyTable[context]
+  if (typeof options === 'string') {
+    options = { target: options }
+  }
+  app.use(proxyMiddleware(context, options))
+})
+
+// handle fallback for HTML5 history API
+app.use(require('connect-history-api-fallback')())
```

Comment:
Why this? We don't use html5 history stuff...

Reply:
I copied it from `script.dev.js`, does it make sense to remove from both, to have the same setup for serving dev and built files?

Reply:
Yes, as long as we remember if we switch to fancy urls in the future (which I would like... just not high priority)

Reply:
Hm, then I'd rather leave it in. It's more of a "minimal change" to me if "yarn dev" and "yarn serve" behave the same.